src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPage.vb

```diff
@@ -631,29 +619,18 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
                 Common.Switches.TracePDPerf("PropPage.Create: Setting the property page's parent")
 
-                Dim ParentControl As Control = Control.FromHandle(hWndParent)
-                Dim AlwaysUseSetParent As Boolean = False
-#If DEBUG Then
-                AlwaysUseSetParent = Common.Switches.PDAlwaysUseSetParent.Enabled
-#End If
-                If ParentControl IsNot Nothing AndAlso Not AlwaysUseSetParent Then
-                    _propPage.Parent = ParentControl
-                    Debug.Assert(_propPage.Parent IsNot Nothing, "Huh?  Deactivate() logic depends on this.")
-                Else
-                    'Not a managed window, use the win32 api method
-                    _hostedInNative = True
-                    ' in order to have scroll bars properly appear in large fonts, wrap
-                    ' the page in a usercontrol (since it supports AutoScroll) that won't
-                    ' scale with fonts. Move(rect) will set the proper size.
-                    Dim sizingParent As New UserControl()
-                    sizingParent.AutoScaleMode = AutoScaleMode.None
-                    sizingParent.AutoScroll = True
-                    sizingParent.Text = "SizingParent" 'For debugging purposes (Spy++)
-                    _propPage.Parent = sizingParent
-                    NativeMethods.SetParent(sizingParent.Handle, hWndParent)
-                    _wasSetParentCalled = True
-                    Debug.Assert(_propPage.Parent Is Nothing OrElse AlwaysUseSetParent, "Huh?  Deactivate() logic depends on this.")
```

Comment:
No, for managed controls it would have been `AlwaysUseSetParent`, which doesn't exist anymore. It was only relevant when we had two code paths.